src/perftest_parameters.h

```diff
@@ -70,6 +70,11 @@
 #include CUDA_PATH
 #endif
 
+#ifdef HAVE_NEURON
+#include <nrt/nrt.h>
+#include <nrt/nrt_experimental.h>
```

Comment:
Do you really want perftest to rely on experimental APIs?
It's probably wise to surround it with a dedicated ifdef..

Reply:
Removed this include entirely.